docs/DevGuide/BuildSystem.md

```diff
@@ -254,6 +254,15 @@ cmake -D FLAG1=OPT1 ... -D FLAGN=OPTN <SPECTRE_ROOT>
   - Multiply the timeout for the respective set of tests by this factor (default
     is `1`).
   - This is useful to run tests on slower machines.
+- SPECTRE_USE_ALWAYS_INLINE
+  - Force SpECTRE inlining (default is `ON`)
+  - If disabled or if the platform is not using a GNU compiler, replaces
+    `SPECTRE_ALWAYS_INLINE` with the `inline` keyword
+  - For SpECTRE build targets that use `SPECTRE_ALWAYS_INLINE`, it may affect
```

Comment:
This bullet is way too hedging. "Toggling this may change things" is how I would summarize it, and while true this gives no guidance as to what I should do with it. You want general guidelines here. Something like "Forced inlining reduces function call overhead and so generally reduces runtime. However, it does increase compile time and compile memory usage. It is also easier to use a debugger when forced inlining is disable. If you are encountering debugger messages like 'function inlined' then forced inlining should be disabled ."

Reply:
Thanks! I definitely had this documentation aimed at active developers, not users.

I wrote essentially the same description for `BLAZE_USE_ALWAYS_INLINE` and `BLAZE_USE_STRONG_INLINE` in my PR that was merged, and I just copied what I did there for `SPECTRE_USE_ALWAYS_INLINE`. Would you like me make the same updates you're suggesting here to the two blaze inlining flags, as well?

Reply:
Sure, that sounds good :)